### DISCUSSION TITLE
Data Augmentation Libraries

### DISCUSSION THREAD

**Discussion by tolgacangoz** [huggingface/diffusers#7078] [General]
Hi `diffusers` community!
`diffusers` codebase mostly uses `torchvision.transforms` for data augmentation. [The V2 transforms are now stable with the 0.17 release for TorchVision](https://github.com/pytorch/vision/releases/tag/v0.17.0). [And, it is recommended to migrate them](https://pytorch.org/vision/stable/transforms.html#v1-or-v2).
Also, [it is argued that `albumentations` is much faster than `torchvision`](https://github.com/albumentations-team/albumentations/issues/765). WDYT?
Or, does this discussion not much relevant in the context of this repo?

**Comment by sayakpaul** [comment]
Yes. Our example scripts tend to be minimal and educational in nature.

**Reply by tolgacangoz** [reply]
OK, got it 👍.
